pymc3/glm/utils.py

```diff
@@ -48,15 +48,15 @@ def any_to_tensor_and_labels(x, labels=None):
     if isinstance(x, pd.DataFrame):
         if not labels:
             labels = x.columns
-        x = x.as_matrix()
+        x = x.values()
```

Comment:
The tests are failing because `values` is a property, so you don't have to call it with the parenthesis. Nevertheless, it's use is discouraged. Pandas recommends using the `to_numpy` method instead. Could you use that instead?

Reply:
so I looked at the wrong outdated pandas doc 🙈